src/SprykerEco/Zed/FactFinderSdk/Persistence/FactFinderSdkQueryContainer.php

```diff
@@ -170,6 +184,8 @@ public function getProductAbstractPriceQuery($concreteProductSku, CurrencyTransf
     }
 
     /**
+     * {@inheritDoc}
+     *
```

Comment:
Please add to the method document block `@module ProductReview`.

Reply:
Fixed.